docs/docs/cmd/viva/engage/engage-community-get.mdx

```diff
@@ -0,0 +1,96 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# viva engage community get
+
+Gets information of a Viva Engage community.
+
+## Usage
+
+```sh
+m365 viva engage community get [options]
+```
+
+## Options
+
+```md definition-list
+`--id <id>`
+: The unique identifier of the community.
+```
+
+<Global />
+
+## Remarks
+
+:::warning
+
+This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+In order to use this command, you need to grant the Azure AD application used by the CLI for Microsoft 365 the permission to the Viva Engage API. To do this, execute the `cli consent --service VivaEngage` command.
```

Comment:
I may not have fully understood this. I have removed the "cli consent" part since it was not included in the issue spec. Please let me know if any other changes are needed.

Reply:
Hi @Saurabh7019, yes the consent part was not needed. For other `viva engage` commands we use the Yammer API which requires this consent. Since this command is built on Graph, we don't have to show this warning here.